src/components/Badges.tsx

```diff
@@ -45,12 +45,12 @@ function Badges() {
           {shouldNotify ? (
             <BadgesHintCard
               text={
-                <>
+                <div>
```

Comment:
`space` isn't recognized for the components unless it's `<div>`, maybe there's a better method